include/libc/stdarg.h

```diff
@@ -1,5 +1,5 @@
-#ifndef STDARG_H
-#define STDARG_H
+#ifndef _STDARG_H
+#define _STDARG_H
```

Comment:
I think i would be better to prefix with `LIBC` like the other headers in this folder
```suggestion
#ifndef LIBC_STDARG_H
#define LIBC_STDARG_H
```

Reply:
I have done this to avoid using libultra's stdarg, since I need _Printf for this and it was complaining about redefined va macros

Should I include libultra's in the sprintf header to avoid this instead? Not sure what's better

Reply:
oh i see, this is part of the printf libultra issue. in that case this is fine, we need to fix this in the future